collector/runtime-benchmarks/asynctest/src/main.rs

```diff
@@ -0,0 +1,50 @@
+use benchlib::benchmark::run_benchmark_group;
+use std::time::Instant;
+use tokio::runtime::Runtime;
+use std::fs::File;
+use std::io::{self, BufRead, BufReader};
+
+async fn read_the_textfile(file_path: &str) -> io::Result<usize> {
+    let file = File::open(file_path)?;
+    let reader = BufReader::new(file);
+    let mut total_characters = 0;
+    for line in reader.lines() {
+        let line = line?;
+        total_characters += line.len();
+    }
+
+    Ok(total_characters)
+}
+
+async fn async_operation() {
```

Comment:
We already have a reasonably large text file with Sherlock Holmes text in the `runtime-benchmarks/data` directory. Check how do other benchmarks use it, and try to reuse that same file.

Reply:
I tried using the decompress_file function, but it returns the output in vec<u8> form and using async on it didn't made much sense, so I used GzDecoder